packages/mask/src/plugins/Savings/protocols/AAVEProtocol.ts

```diff
@@ -0,0 +1,311 @@
+    public type = ProtocolType.AAVE
+    public name = 'AAVE'
+    public image = 'aave'
+    public base = 'ETH'
+    public pair = 'aAAVE'
+    public decimals = 18
+    public apr = '0.00'
+    public balance = new BigNumber('0')
+    public availableNetworks: SavingsNetwork[] = [
+        {
+            chainId: ChainId.Mainnet,
+            chainName: 'Ethereum',
+            contractAddress: getSavingsConstants(ChainId.Mainnet).AAVE || ZERO_ADDRESS,
+        },
+        {
+            chainId: ChainId.Kovan,
+            chainName: 'Kovan',
+            contractAddress: getSavingsConstants(ChainId.Kovan).AAVE || ZERO_ADDRESS,
+        },
+    ]
+
+    public getFungibleTokenDetails(chainId: ChainId): FungibleTokenDetailed {
+        let contractAddress = ''
+
+        for (const network of this.availableNetworks) {
+            if (network.chainId === chainId) {
+                contractAddress = network.contractAddress
+            }
+        }
+
+        return {
+            type: 1,
+            chainId: chainId,
+            address: contractAddress,
+            symbol: 'aAAVE',
+            decimals: 18,
+            name: 'AAVE Interest Bearing AAVE',
+            logoURI: [
+                // 'https://static.debank.com/image/eth_token/logo_url/0xae7ab96520de3a18e5e111b5eaab095312d7fe84/f768023f77be7a2ea23c37f25b272048.png',
+                'https://tokens.1inch.io/0xffc97d72e13e01096502cb8eb52dee56f74dad7b.png',
+            ],
+        }
+    }
+
+    public async getApr(chainId?: ChainId) {
+        try {
+            const subgraphUrl = getSavingsConstants(chainId ?? ChainId.Kovan).AAVE_SUBGRAPHS || ''
+            const body = JSON.stringify({
+                query: `{
+                reserves (where: {
+                    underlyingAsset: "${getSavingsConstants(chainId ?? ChainId.Kovan).AAVE || ZERO_ADDRESS}"
+                }) {
+                    id
+                    name
+                    underlyingAsset
+                    price {
+                     id
+                    }
+                    liquidityRate
+                    }
+                }`,
+            })
+            const response = await fetch(subgraphUrl, {
+                method: 'POST',
+                headers: { 'Content-Type': 'application/json' },
+                body: body,
+            })
+            const fullResponse = await response.json()
+            const liquidityRate = +fullResponse.data.reserves[0].liquidityRate
+
+            const RAY = 10 ** 27 // 10 to the power 27
+            const SECONDS_PER_YEAR = 31536000
+            // APY and APR are returned here as decimals, multiply by 100 to get the percents
+            const apr = liquidityRate / RAY
+            this.apr = apr.toFixed(2)
+            return apr.toFixed(2)
+        } catch (error) {
+            console.log('AAVE `getApr()` error', error)
+            // Default APR
+            this.apr = '0.17'
+            return '0.17'
```

Comment:
```suggestion
            this.apr = DEFAULT_APR
            return DEFAULT_APR.toFixed(2)
```

`apr` data type use `BigNumber` replace it

Reply:
Can you point to where BigNumber is being referenced?
The interface return string representation of big number as best as i can see

Reply:
Done